blog/thirdweb.md

```diff
@@ -0,0 +1,260 @@
+---
+slug: thirdweb-wallet-upload
+hide_table_of_contents: false
+title: "Introducing Wallet SDK and Sending remote storage"
+date: 2023-05-17
+authors: fabri
+image: "https://blog.thirdweb.com/content/images/size/w2000/2023/05/How-to-use-smart-wallet-.png"
+description: "Sending remote attachments with XMTP"
+tags:
+- Content Types
+- SDKs
+- Media
+- Developers
+- WalletSDK
+- Thirdweb
+---
+
+Creating an effective 'Connect Wallet' flow is a challenging aspect of web3 development. Thankfully, the team at Thirdweb has developed an excellent SDK, simplifying this process and enabling a superior wallet experience for your users.
+
+<!--do we want to request a blog card visual from Nad?-->
+
+<!--truncate-->
```

Comment:
We insert this comment to tell docusaurus where to stop displaying content on the blog preview page: https://junk-range-possible-git-thirdweb-blog-post-xmtp-labs.vercel.app/blog